### PR TITLE
Create 2017-02-28-photography.markdown

### PR DESCRIPTION
**Pull request by rampatra**
New jekyll theme added.

### CODE REVIEW

**Comment by ramswaroop** [comment]
I guess github has some issues today with uploading of files. I am not able to upload my thumbnail.

File: _posts/2017-02-28-photography.markdown [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+title: Photography
+homepage: https://github.com/ramswaroop/photography
+download: https://github.com/ramswaroop/photography/archive/master.zip
+demo: http://photography.ramswaroop.me/
+author: Ram Patra
+thumbnail: photography.jpg
```
Comment (hilderoters) [inline _posts/2017-02-28-photography.markdown]: ## # ###

Comment (ramswaroop) [inline _posts/2017-02-28-photography.markdown]: Hey, I didn't get the comment. Can you please elaborate?

**Comment by ramswaroop** [comment]
Are there any updates on this?

**Comment by junlulocky** [comment]
@mattvh @gynter @InsidiousMind @jarrekk - would you please have a look at the demo & check whether it is suitable to accept?  
@ramswaroop - I would recommend you to push it to https://github.com/jekyller/jekyllplugins, which is a jekyll plugin host rather than a whole theme.

Thanks.

**Pull request closed without merging**